README.md

```diff
@@ -49,6 +50,21 @@ To install, simply add the following line to your Podfile:
 ```ruby
 pod "WistiaKit"
 ```
+### Carthage
+Starting from `0.30.2` WistiaKit should work with Carthage, sample Cartfile:
+
+```ruby
+github "Wistia/Wistiakit" ~> 0.30.2
+```
+At this moment this creates 2 frameworks `WistiaKit` and `WistiaKitCore`.
+In the future this could be converted into one framework, but for now this is what we have, and its working!
```

Comment:
Let's explain the rationale behind the two frameworks (can simply reference the following section, "Usage", where they are explained in detail) and drop this line.